hera_cal/io.py

```diff
@@ -128,6 +128,28 @@ def update(self, gains=None, flags=None, quals=None, total_qual=None):
                 self.total_quality_array[0, :, :, ip] = total_qual[pol].T
 
 
+def get_blt_slices(uvo):
+    '''For a pyuvdata-style UV object, get the mapping from antenna pair to blt slice.
+    
+    Arguments:
+        uvo: a "UV-Object" like UVData or baseline-type UVFlag
```

Comment:
Just a suggestion, but it might be useful to put in a check that the input adheres to this. Otherwise it might be pretty difficult to diagnose issues.

Reply:
I'm not sure how to do that without unnecessarily restricting what can go into the function (who knows what objects we'll build in the future that could use this method). It's probably a "for experts" method, so I'd consider making it `_get_blt_slices`, but otherwise I think it's OK.